src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -217,31 +217,19 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
   /**
    * Creates a set of JES inputs for a backend call.
+   * Note that duplicates input files (same gcs path) will be (re-)localized every time they are referenced.
```

Comment:
Just wanted to bring some attention on this, I don't know if this is expected behavior or should be addressed ?